### DISCUSSION TITLE
How to live stream the webrtc url in a videojs player

### DISCUSSION THREAD

**Discussion by bindu1114** [bluenviron/mediamtx#1479] [Questions]
Thank you for all this amazing work. I am trying to connect the webrtc live stream to videojs player instead of iframe, but it does not work. Please provide suggestions on how this can be achieved. Thank u

**Comment by aler9** [comment]
Hello, video.js only supports WebRTC streams coming from Ant media server, through a custom plugin. You can either switch to HLS (which is supported by video.js) or drop video.js and use an embedding solution like an iframe or the javascript class, as described here:
https://github.com/aler9/mediamtx#embedding-1

[a comment by an automated account is not shown]
